tests/xharness/Jenkins/TestSelector.cs

```diff
@@ -80,7 +81,7 @@ class TestSelector {
 		static readonly string [] dotnetPrefixes = {
 			"dotnet",
 			"msbuild",
-			"tests/dotnet",
+			".*dotnet.*",
```

Comment:
there's a `"dotnet"` entry two lines above
the variable is also misnamed/confusing now since it's not just prefixes anymore